### DISCUSSION TITLE
auto discharge - what is the method?

### DISCUSSION THREAD

**Discussion by TorinoSM** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1257] [Q&A] [answered]
Is it a natural discharge due to running applications or is a special Al Dente functionality used that increases processor consumption during the discharge period?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @TorinoSM,

It is the "natural" discharge of running applications. AlDente is not artificially increasing the load.

Best,
Matthias

**Reply by TorinoSM** [reply]
thanks a lot!
regards
